localisation/00_memewc_united_localisation.csv

```diff
@@ -1,6 +1,18 @@
 #CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
 RULE_GROUP_WARCRAFT_ARTIFACTS;WarCraft - Artifacts;;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+RULE_PHYSICAL_WEAPON_CLASS_REQUIREMENT;�RWarCraft�!: Class Requirements for Physical Weapons;;;;;;;;;;;;;x
+RULE_PHYSICAL_WEAPON_CLASS_REQUIREMENT_DEFAULT_DESC;Only physical classes can use some physical weapons.;;;;;;;;;;;;;x
+RULE_PHYSICAL_WEAPON_CLASS_REQUIREMENT_UNRESTRICTED_DESC;Adult character of any class can use some physical weapons.;;;;;;;;;;;;;x
+RULE_RANGED_WEAPON_CLASS_REQUIREMENT;�RWarCraft�!: Class Requirements for Ranged Weapons;;;;;;;;;;;;;x
+RULE_RANGED_WEAPON_CLASS_REQUIREMENT_DEFAULT_DESC;Only physical classes can use some ranged weapons.;;;;;;;;;;;;;x
```

Comment:
Removed `some`, because I don't _think_ its necessary. 
```suggestion
RULE_RANGED_WEAPON_CLASS_REQUIREMENT_DEFAULT_DESC;Only physical classes can use ranged weapons.;;;;;;;;;;;;;x
```